charts/garnet/templates/cmd-pod.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cmd
+  namespace: garnet
+  labels:
+    app: cmd
+spec:
+  containers:
+  - name: cmd
+    image: redis:latest
```

Comment:
Why does this say redis-latest? We have nothing to do with that image.

Reply:
that pod is used for testing. So, i'll remove it from the pr.

Reply:
Also to use the redis-cli, don't we have to use the redis image?

Reply:
Isn't redis-tools sufficient for this?